modules/op-creating-mounting-kms-authentication-token-secret.adoc

```diff
@@ -0,0 +1,56 @@
+// This module is included in the following assemblies:
+// * secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-mounting-kms-authentication-token-secret_{context}"]
+= Creating and mounting the KMS authentication token secret
+
+You can provide the authentication token for the KMS server (the value of `VAULT_TOKEN`) using a secret. You must create this secret, mount it on the {tekton-chains} controller, and set the `signers.kms.auth.token-dir` parameter to the directory where the secret is mounted.
+
+.Prerequisites
+
+* You installed the `oc` command-line utility.
+* You are logged in to your {OCP} claster with administrative rights for the `tekton-chains` namespace.
+
+.Procedure
+
+. Create a secret named `kms-secrets` with the `KMS_AUTH_TOKEN` file that contains the authentication token for the KMS server by entering the following command:
++
+[source, terminal]
+----
+$ oc create secret generic kms-secret -n tekton-chains \ #
```

Comment:
secret name "kms-secret" mismatches with secret name in config section. Please set it to "kms-secrets" so that it is consistent.

Reply:
The secret name can be anything not just `kms-secret` but user need to make sure that same secretName is added in the chains controller deployment

Reply:
This comment on naming was in the context of example spec from line number 28-55. Just to ensure consistency for the reader to relate to the example. Otherwise name can be anything

Reply:
I made the name consistent as kms-secrets - I want to keep the procedure as simple as possible so would want to avoid explaining the part about other names